numpy/core/tests/test_arrayprint.py

```diff
@@ -555,6 +555,25 @@ def test_linewidth_repr(self):
                    2, 2, 2, 2])""")
         )
 
+        a = np.full(8, fill_value=2)
+
+        np.set_printoptions(linewidth=18, legacy=False)
+        assert_equal(
+            repr(a),
+            textwrap.dedent("""\
+            array([2, 2, 2,
+                   2, 2, 2,
+                   2, 2])""")
```

Comment:
I'm guessing we prefer this output to the following:
```
array([2, 2, 2, 2,
       2, 2, 2, 2
       ])
```

Reply:
I say yes !